servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcRouter.java

```diff
@@ -952,5 +849,31 @@ public Completable closeAsync() {
         public Completable closeAsyncGracefully() {
             return closeable.closeAsyncGracefully();
         }
+
+        private static AsyncCloseable toAsyncCloseable(final GracefulAutoCloseable original) {
```

Comment:
did you consider putting this someone more visible (`AutoClosableUtils`, `AsyncCloseables`, ..)?

Reply:
I will do it in a separate PR for visibility and will use the same utility in HTTP conversions